bin/BuildPackages.sh

```diff
@@ -51,6 +51,7 @@ while [[ "$#" -ge 1 ]]; do
 
     --no-strict)      STRICT=no ;;
     --strict)         STRICT=yes ;;
+    --add-package-config-*)   typeset PACKAGE_CONFIG_ARGS_${option:21}="$1"; shift ;;
```

Comment:
What effect does `typeset` have if no type restriction is specified?

Reply:
Otherwise, I get an error.
```
$ bash ../bin/BuildPackages.sh --add-package-config-semigroups="--with-external-planarity"
../bin/BuildPackages.sh: line 54: PACKAGE_CONFIG_ARGS_semigroups=--with-external-planarity=: command not found
```

Reply:
OK, then I guess it's needed because of the computed name of the variable?!? I am simply trying to understand the code. But ah well, if it works ... *shrug*

Reply:
Yes, it's because of the computed name.